bin/aurelia.js

```diff
@@ -8,7 +8,7 @@ process.env.INIT_CWD = process.cwd();
 var DEV_ENV = parseInt(process.env.AURELIA_CLI_DEV_ENV, 10);
 
 const cli = new Liftoff({
-  name: 'aurelia',
+  name: 'aurelia-cli',
```

Comment:
@ahmedshuhel Liftoff wants the name of the package in node_modules `aurelia-cli` is the correct name